### PR TITLE
Handle Events For Item List Music

### PR DESCRIPTION
**Pull request by VanHieu7119**
Handle events for items List Music
Link Redmine: https://edu-redmine.framgia.vn/issues/2699

### CODE REVIEW

**Comment by VanHieu7119** [comment]
ready

File: app/src/main/java/com/framgia/music_39/screen/listmusic/ListMusicFragment.java [hunk 1]
```diff
@@ -17,11 +18,13 @@
...
 
 public class ListMusicFragment extends Fragment
-        implements ListMusicContract.View, ItemClickListener {
+        implements ListMusicContract.View, ItemClickListener, View.OnClickListener {
 
     private static String ARGUMENT_GENRE = "ARGUMENT_GENRE";
 
     private ListMusicAdapter mListMusicAdapter;
+    private ImageButton mButtonBack;
+    private View mView;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/music_39/screen/listmusic/ListMusicFragment.java]: chuyển thành biến local

File: app/src/main/res/drawable/ic_back_24dp.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="32dp" android:tint="#6E6E6E"
+    android:viewportHeight="24.0" android:viewportWidth="24.0"
```
Comment (tuanlvt-1117) [inline app/src/main/res/drawable/ic_back_24dp.xml]: reformat code

**Comment by tuanlvt-1117** [comment]
commented

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Comment by VanHieu7119** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Comment by tuanlvt-1117** [comment]
merged

**Pull request merged**